engine/src/hopeit/server/web.py

```diff
@@ -93,6 +93,7 @@ def prepare_engine(*, config_files: List[str], api_file: Optional[str], start_st
         logger.info(__name__, f"Loading app config file={config_file}...")
         config = _load_app_config(config_file)
         config.server = server_config
+        config.groups = groups
```

Comment:
Mutating the config is a bad practice. the case were we plug the "server" config into the AppConfig can be also argibily but the use case for that situation is to compose a comfiguration from an app config and server config file. It does not apply for groups.

I will keep a list of `enabled_groups` instead at runtime probably in the AppEngine object.